k0799343/quiz1.py

```diff
@@ -0,0 +1,38 @@
+############################
+#
+# Please complete this quiz
+#
+
+# declare an int variable named total with the value of 0
+total = 0
+# declare an array named numbers with all numbers 0 - 20
+numbers = [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20]
+# write a for loop that processes each number
+
+for num in numbers:
+    if num == (2,4,6,8,10,12,14,16,18,20):
+        num == even
+        
+    elif num == (1,3,5,7,9,11,13,15,17,19):
+        num == odd
+
+def even():
```

Comment:
I think it would help to separate out the thing to do from the question being asked... for example in response to the question is this number even we are supposed to add it to total

Separate out the concepts for asking the question from what to do... when I see something like even I expect it to tell me if a number is even or not, so try something like

```
def even(num):
    #do something here to return True or False
```

Solve that problem so that inside your loop you can do something different based on the answer to this function, like print even or print odd.

Once you've got that working, you can go on to the next step and instead of printing even, add it to total.